src/AccessLog.php

```diff
@@ -115,27 +104,45 @@ public function process(ServerRequestInterface $request, DelegateInterface $dele
     }
 
     /**
+     * Generates and returns the message format
+     *
+     * @return string
+     */
+    private function getFormat()
+    {
+        $message = $this->vhost ? '%v' : '%h';
```

Comment:
I see. I'm interested in `vhost_combined` format defined in debian package: https://anonscm.debian.org/cgit/pkg-apache/apache2.git/tree/debian/config-dir/apache2.conf?h=jessie#n206